### PR TITLE
security 

### PR DESCRIPTION
**Pull request by calebrwalk5**

### CODE REVIEW

File: website/backend.js [hunk 1]
```diff
@@ -26,7 +25,11 @@ async function init(app) {
     app.use(express.static('src/static'));
     app.use('/assets', express.static(__dirname + 'public/assets'));
     app.use('/static', express.static(__dirname + 'src/static/assets'));
-    app.use(requestIp.mw());
+    app.use((req, res, next) => {
+        const clientIp = req.headers['x-forwarded-for'] || req.connection.remoteAddress;
+        console.log(`Client IP: ${clientIp}`);
```
Comment (Itz-Hyperz) [inline website/backend.js]: if(config.debugMode) console.log(`Client IP: ${clientIp}`);

Comment (calebrwalk5) [reply website/backend.js]: smart man

**Comment by calebrwalk5** [comment]
like i said @Itz-Hyperz  i am an idea machine, i give good ideas not code

**Comment by Itz-Hyperz** [comment]
not entirely sure mysqldump.on('error') would ever be emitted, but just in case, and to make you feel better, out of pity, I will merge :]

**Pull request merged**